users/forms.py

```diff
@@ -24,3 +25,18 @@ class Meta:
         model = CustomUser
         fields = '__all__'
         widgets = {'country': CountrySelectWidget()}
+
+
+class LoginAuthentication(AuthenticationForm):
+    def clean(self):
+        username = self.cleaned_data.get('username')
+        if "@" not in username:
+            username = username + "@codepoets.it"
+        password = self.cleaned_data.get('password')
+
+        if username is not None and password:
```

Comment:
I think there is something funky going on with the data types. If `username` will be `None`, then the line https://github.com/Code-Poets/sheetstorm/pull/59/files#diff-ee6dfbb85387b31ef4a4e46efdb9be53R33 will fail:

```
>>> '@' not in None
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: argument of type 'NoneType' is not iterable
```

I think the quickest solution for this is to move condition `if "@" not in username:` below this condition.